include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -162,6 +162,8 @@ class HexagonGame
     Utils::LuaMetadata luaMetadata;
 
     // Lua related methods
+    void redefineLuaFunctions();
+
```

Comment:
```suggestion
    void redefineLuaFunctions();
```

Reply:
I really do love this nitpick of yours xd